tests/test_stac.py

```diff
@@ -74,7 +73,8 @@ def test_incorrect_metadata_txt(self):
 
         self.assertEqual(item.id, "al_m_3008501_ne_16_1_20110815")
         self.assertEqual(
-            item.datetime, str_to_datetime("20110815") + timedelta(hours=16)
+            item.datetime,
+            datetime.strptime("2011-08-15T16:00:00Z", "%Y-%m-%dT%H:%M:%SZ"),
```

Comment:
For future reference: https://docs.python.org/3.8/library/datetime.html#datetime.datetime.fromisoformat
(which on python < 3.11 (or 3.10?) requires a timezone of `+00:00` instead of `Z`